doc/bitcoin-conf.md

```diff
@@ -49,4 +49,12 @@ rpcport=4000
 
 The configuration file is not automatically created; you can create it using your favorite text editor. By default, the configuration file name is `bitcoin.conf` and it is located in the Bitcoin data directory, but both the Bitcoin data directory and the configuration file path may be changed using the `-datadir` and `-conf` command-line options.
 
+### Default configuration file locations
+
+| Operating System | Data Directory | Path |
+| --- | --- | --- |
+| Windows | %APPDATA%\Bitcoin\ | *C:\Users\username\AppData\Roaming\Bitcoin\bitcoin.conf* |
```

Comment:
Nit: Could drop *italics* for the path to make the back slashes look slightly less like forward slashes? :-)